astropy/tests/tests/test_run_tests.py

```diff
@@ -10,6 +10,13 @@
 
 import pytest
 
+try:
+    import pytest_doctestplus
+except ImportError:
+    PYTEST_DOCTESTPLUS_INSTALLED = False
+else:
+    PYTEST_DOCTESTPLUS_INSTALLED = True
+
```

Comment:
Personally I think this would be cleaner with a `pytest.importorskip('pytest_doctestplus')` inside `test_doctest_float_replacement`.